core/src/test/java/jenkins/util/io/PathRemoverTest.java

```diff
@@ -418,6 +423,34 @@ public void testForceRemoveRecursive_ParentIsSymbolicLink() throws IOException {
         assertFalse(d1.exists());
     }
 
+    @Test
+    @Issue("JENKINS-55448")
+    public void testForceRemoveRecursive_AbortsAfterTooManyErrors() throws IOException {
+        assumeTrue(Functions.isWindows());
```

Comment:
I'm not sure how to test this behavior on Linux without a ton of mocking, which I prefer to avoid, so it's Windows-only.